### PR TITLE
python script for turning paper metadata to yaml file

### PR DESCRIPTION
**Pull request by cimeister**

### CODE REVIEW

File: aclpub2/paper_csv_to_yaml.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+all_entries = []
+with open(args.csv_path, 'r') as f:
+    reader = csv.reader(f, delimiter=',')
+    header = next(reader)
+    for row in reader:
+        row_out = collections.OrderedDict()
+        authors = collections.defaultdict(dict)
+        for k, v in zip(header, row):
```
Comment (zhzhang) [inline aclpub2/paper_csv_to_yaml.py:31]: @cimeister could you change this to use https://docs.python.org/3/library/csv.html#csv.DictReader ? it'll make it slightly more readable I think.

**Review by zhzhang — CHANGES_REQUESTED** [review]

File: aclpub2/paper_csv_to_yaml.py (hunk 1 above)
Comment (cimeister) [reply aclpub2/paper_csv_to_yaml.py]: good idea!

**Comment by zhzhang** [comment]
Btw, is this for softconf or openreview metadata? If so, there are folders in the repo for scripts pertaining to those.
Would you be able to push up a snippet of example input to this script as well?

**Comment by cimeister** [comment]
> 

I believe this is for the softconf data. @ryancotterell shared a google sheet with me. I'll add a snippet.

**Comment by zhzhang** [comment]
Ah ok! Last change would be to move the script under `softconf/` and move the example snippet into `examples/softconf/`.
Thanks so much!

**Review by zhzhang — APPROVED** [review]

**Pull request merged**